docs/references/openapi-admin.yaml

```diff
@@ -3,7 +3,7 @@ openapi: "3.0.2"
 info:
   title: Certifaction Admin API
   description: |
-    Certifaction Admin API is used to to manage users, roles and organisations.
+    Certifaction Admin API is used to manage users, roles, teamspaces, and cost centers.
```

Comment:
Cost center is not in FE yet and untested. Can you comment it out for now?

Reply:
I tested the endpoints via cURL and checked that entries were updated/removed but can comment it out

Reply:
I guess it's also used in other places to charge, will comment it out :)

Reply:
[803cfb4](https://github.com/certifaction/cli/pull/45/commits/803cfb4e0802bbb7039beae819d3bb27114d603e)